app/controllers/util/CustomMetadataUtils.scala

```diff
@@ -72,15 +72,27 @@ class CustomMetadataUtils(allCustomMetadataProperties: List[CustomMetadata]) {
       case Text =>
         property.propertyType match {
           case Defined =>
-            DropdownField(
-              property.name,
-              fieldLabel,
-              fieldDescription,
-              property.multiValue,
-              property.values.sortBy(_.uiOrdinal).map(v => InputNameAndValue(v.value, v.value)),
-              property.defaultValue.map(value => InputNameAndValue(value, value)),
-              isRequired
-            )
+            if (property.multiValue) {
+              MultiSelectField(
+                property.name,
+                fieldLabel,
+                fieldDescription,
+                property.multiValue,
+                property.values.sortBy(_.uiOrdinal).map(v => InputNameAndValue(v.value, v.value)),
```

Comment:
This value is used for either option, so maybe abstract out to a val? `val options = property.values.sortBy(_.uiOrdinal).map(v => InputNameAndValue(v.value, v.value))`